### DISCUSSION TITLE
achievement

### DISCUSSION THREAD

**Discussion by Abeir01** [drknzz/GitHub-Achievements#8] [Q&A] [answered]
how to get achievements

**Comment by drknzz** [comment]
You have to answer at least 2 discussions and get 2 accepted answers.

You can try answering [this discussion](https://github.com/drknzz/GitHub-Achievements/discussions/9), so I can mark your answer as accepted.

**Comment by Santana19996** [comment]
Badges for me pleaseee

**Reply by drknzz** [reply]
There you go :octocat:

**Comment by TobiasMelkersson** [comment, marked as answer]
Brain badge plz!
